compose/cpp-kgs/Dockerfile

```diff
@@ -0,0 +1,160 @@
+# To build, navigate to the *root* of this repo and run:
```

Comment:
Is it possible to do away with the duplication of code between the two Dockerfiles?
Either via some method of including another Dockerfile or via just mounting the necessary neural net files / configs at runtime?

Reply:
You can do `FROM humancompatible/goattack:cpp` to build on top of it (will require building that one first, but that should be OK). You could also add it as a separate build stage to `compose/cpp/Dockerfile` but that seems worse (we probably want to keep KGS fairly separated).

Reply:
Adam's suggestion to build on top of the base one worked, implemented now.